src/java.base/share/classes/java/io/ObjectStreamConstants.java

```diff
@@ -35,108 +35,108 @@ public interface ObjectStreamConstants {
     /**
      * Magic number that is written to the stream header.
      */
-    static final short STREAM_MAGIC = (short)0xaced;
+    short STREAM_MAGIC = (short)0xaced;
```

Comment:
I'd prefer to retain the `static`, it is easier to read and not have to remember that this declaration is in an interface.

Reply:
I think the class should not be an interface in the first place but this is hard to change now.

Reply:
Just because the static is not required does NOT mean it should be omitted!.